spring_week1/src/main/java/prgms/spring_week1/CommandLine.java

```diff
@@ -0,0 +1,90 @@
+package prgms.spring_week1;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import prgms.spring_week1.domain.customer.service.CustomerService;
+import prgms.spring_week1.domain.voucher.model.Voucher;
+import prgms.spring_week1.domain.voucher.model.type.VoucherType;
+import prgms.spring_week1.domain.voucher.repository.VoucherRepository;
+import prgms.spring_week1.domain.voucher.service.VoucherService;
+import prgms.spring_week1.exception.EmptyListException;
+import prgms.spring_week1.exception.NoSuchOptionValue;
+import prgms.spring_week1.exception.NoSuchVoucherType;
+import prgms.spring_week1.io.Input;
+import prgms.spring_week1.io.Output;
+import prgms.spring_week1.menu.Menu;
+
+@Component
+public class CommandLine implements Runnable{
+    private final Input input;
+    private final Output output;
+    private final VoucherRepository voucherRepository;
+    private final VoucherService voucherService;
+    private final CustomerService customerService;
+    private boolean IS_RUNNING = true;
+    private static final Logger log = LoggerFactory.getLogger(CommandLine.class);
+
+    public CommandLine(Input input, Output output, VoucherRepository voucherRepository, VoucherService voucherService, CustomerService customerService) {
+        this.input = input;
+        this.output = output;
+        this.voucherRepository = voucherRepository;
+        this.voucherService = voucherService;
+        this.customerService = customerService;
+    }
+
+    @Override
+    public void run() {
+        while (IS_RUNNING) {
+            output.printMenuList();
+            String selectOption = input.inputTextOption();
+            switch (findMenuName(selectOption)) {
+                case EXIT -> System.exit(0);
+                case CREATE -> selectVoucherType();
+                case LIST -> printAllVoucher();
+                case BLACK -> output.printBlackConsumerList(customerService.blackConsumerList());
+                default -> output.printWrongMenuMessage();
+                }
+            }
+        }
```

Comment:
코드 정렬 (띄어씌기)가 적용 되어 있지 않은것 같아요.
아래의 코드를 제안 드려보았어요.


```suggestion
            }
        }
    }
```